src/main/resources/views/CommercialVehicles/LEC_FAIL.hbs

```diff
@@ -9,38 +9,49 @@
 <body>
     <div class="header-block">
         <div class="header-left">
-            <h2>Transport for London</h2>
-            <h3 class="serial-number">Serial number: <span id="serialNumber">{{lecData.serialNumber}}</span></h3>
+            <h2 class="tfl-title">Transport for London</h2>
+            <h3 class="serial-number">Serial number LF: <span id="serialNumber">{{lecData.serialNumber}}</span></h3>
         </div>
         <div class="header-right">
-            <div> LOGO </div>
-            <p> DATE?? </p>
+            <div class="tfl-logo-container"><img class="tfl-logo" src="{{root}}/assets/images/TFL_roundel.png"/></div>
+            <p class="header-date"><strong>LEC1 OCTOBER 2019</strong></p>
```

Comment:
Do you need to add a variable here?

Reply:
Apparently not. This is a Certificate version number, rather than a date.